manifests/velero-1.7-upgrade.yaml

```diff
@@ -0,0 +1,40 @@
+#
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+apiVersion: manifests/v1beta1
+metadata:
+  name: velero-double-jump
+spec:
+  sources:
+    charts:
+    - name: csm
+      type: repo
+      location: https://arti.dev.cray.com/artifactory/csm-helm-stable-local/
+    - name: csm-algol60
+      type: repo
+      location: https://artifactory.algol60.net/artifactory/csm-helm-charts/
+  - name: cray-velero
```

Comment:
This is not valid yaml/loftsman manifest. 
```bash
$ yq e . velero-1.7-upgrade.yaml
Error: bad file 'velero-1.7-upgrade.yaml': yaml: line 4: did not find expected key
```

The `charts` keys is missing. Should look something like the following:
```yaml
apiVersion: manifests/v1beta1
metadata:
  name: velero-double-jump
spec:
  sources:
    charts:
    - name: csm
      type: repo
      location: https://arti.dev.cray.com/artifactory/csm-helm-stable-local/
    - name: csm-algol60
      type: repo
      location: https://artifactory.algol60.net/artifactory/csm-helm-charts/
  charts:
  - name: cray-velero
    source: csm-algol60
    version: 1.7.1
    namespace: velero
```